lib/core/seleniumRunner.js

```diff
@@ -160,7 +161,7 @@ class SeleniumRunner {
         'for page complete check script to return true',
         function (d) {
           return d
-            .executeScript(pageCompleteCheck, waitTime)
+            .executeScript(pageCompleteCheck, { waitTime, maxLoadTime })
```

Comment:
It would be cool to see the `maxLoadTime` used in all checks since 2 minutes seems really long for any pageload no matter how you check the end of it. It might be easy to do that by having a separate `maxLoadTime` check just before this check.

Reply:
Yep, let me fix that, good idea.